source/views/sis/course-search/lib/build-filters.js

```diff
@@ -18,6 +18,7 @@ export async function buildFilters(): Promise<FilterType[]> {
 
 	const allGEs = ges.map(ge => ({title: ge}))
 	const allDepartments = departments.map(dep => ({title: dep}))
+	const courseLevelOptions = [{title: 100}, {title: 200}, {title: 300}]
```

Comment:
Can you make these strings?

Reply:
sure

Reply:
Just seems odd to allow non-string Title properties